src/components/SpaceProposalVotesModal.vue

```diff
@@ -0,0 +1,238 @@
+
+watch(
+  () => [props.open, bottom.value],
+  to => {
+    if (props.open && to) {
+      loadMore(loadMoreVotes);
+    }
+  },
+  {
+    immediate: true
+  }
+);
+
+watch(
+  () => props.votes,
+  to => {
+    if (to.length > 0) {
+      votes.value = [...to];
+    }
+  }
+);
+
+watch(
+  () => props.open,
+  to => {
+    if (!to) {
+      votes.value = [...props.votes];
+    }
+  }
+);
+</script>
+
+<template>
+  <BaseModal
+    :open="open"
+    max-height="500px"
+    max-width="700px"
+    @close="$emit('close')"
+  >
+    <template #header>
+      <div
+        class="flex flex-row content-center items-center justify-center gap-x-4 pl-4"
+      >
+        <h3>{{ $t('votes') }}</h3>
+        <BaseSearch
+          :model-value="voteQuery"
+          :placeholder="$t('searchPlaceholder')"
+          :autofocus="true"
+          class="flex-auto pr-2"
+        />
+      </div>
+    </template>
+    <div class="flex w-full flex-col">
+      <div
+        ref="votesScrollWrapper"
+        class="flex h-full flex-col overflow-auto"
+        :style="{ maxHeight: '450px' }"
+      >
+        <div
+          v-for="(vote, i) in sortedVotes"
+          :key="i"
+          :class="[
+            'flex items-center border-t px-4 py-[14px]',
+            { '!border-0': i === 0 }
+          ]"
+          :data-testid="`proposal-votes-list-item-${i}`"
+        >
+          <BaseUser
+            :key="vote.voter"
+            :profile="profiles[vote.voter]"
+            :address="vote.voter"
+            :space="space"
+            :proposal="proposal"
+            width-class="w-[110px] min-w-[110px] xs:w-[130px] xs:min-w-[130px] text-left"
+          />
+
+          <SpaceProposalVotesListItemChoice :proposal="proposal" :vote="vote" />
+          <div
+            class="flex w-[110px] min-w-[110px] items-center justify-end whitespace-nowrap text-right text-skin-link xs:w-[130px] xs:min-w-[130px]"
+          >
+            <span
+              v-tippy="{
+                content: vote.scores
+                  ?.map(
+                    (score, index) =>
+                      `${formatCompactNumber(score)} ${titles[index]}`
+                  )
+                  .join(' + ')
+              }"
+            >
+              {{
+                `${formatCompactNumber(vote.balance)} ${shorten(
+                  proposal.symbol || space.symbol,
+                  'symbol'
+                )}`
+              }}
+            </span>
+            <BaseButtonIcon @click="openReceiptModal(vote.ipfs)">
+              <BaseIcon name="signature" />
+            </BaseButtonIcon>
+            <BaseButtonIcon
+              v-if="vote.reason !== '' && props.proposal.privacy !== 'shutter'"
+              v-tippy="{
+                content: vote.reason
+              }"
+              class="cursor-default p-0"
+            >
+              <i-ho-annotation class="text-[16px]" />
+            </BaseButtonIcon>
+          </div>
+        </div>
```

Comment:
This is also again repeating code that we already have. We should refactor this into a component `SpaceProposalVotesListItem` and use it both here and in `SpaceProposalVotesList`.